recipes/wg21-linear_algebra/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class LAConan(ConanFile):
+    name = "wg21-linear_algebra"
+    homepage = "https://github.com/BobSteagall/wg21"
+    description = "Production-quality reference implementation of P1385: A proposal to add linear algebra support to the C++ standard library"
+    topics = ("linear-algebra", "multi-dimensional", "maths")
+    license = "BSD-3-Clause"
```

Comment:
https://spdx.org/licenses/NCSA.html

```suggestion
    license = "NCSA"
```